android/include/mlops-agent-interface.h

```diff
@@ -0,0 +1,180 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/**
+ * @file    mlops-agent-interface.h
+ * @date    5 April 2023
+ * @brief   A set of exported ml-agent interfaces for managing pipelines, models, and other service.
+ * @see     https://github.com/nnstreamer/deviceMLOps.MLAgent
+ * @author  Wook Song <wook16.song@samsung.com>
+ * @bug     No known bugs except for NYI items
+ */
+
+#ifndef __MLOPS_AGENT_INTERFACE_H__
+#define __MLOPS_AGENT_INTERFACE_H__
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
+
+#include <stdint.h>
+
+/**
+ * @brief An interface exported for setting the description of a pipeline.
+ * @param[in] name A name indicating the pipeline whose description would be set.
+ * @param[in] pipeline_desc A stringified description of the pipeline.
+ * @return 0 on success, a negative error value if failed.
+ */
+int ml_agent_pipeline_set_description (const char *name, const char *pipeline_desc);
```

Comment:
I suggest adding jni folder and using common header for android build.
We don't need to change meson for android.
```
jni/mlops-agent.mk
    MLOPS_AGENT_SRCS := $(MLOPS_AGENT_ROOT)/daemon/mlops-agent-android.c
```

Then, including MLOPS_AGENT_SRCS when building android library in api repo.

Reply:
Honestly, I don't 100% understand it. I'll study and try it.